backend/api/schemas/pipeline.py

```diff
@@ -39,6 +49,14 @@ class PipelineNodeCreate(BaseModel):
     container_is_input: bool
     container_is_output: bool
     dicom_node_id: Optional[int]
+    conditions: Optional[List[PipelineNodeConditionCreate]] = []
+
+    @root_validator
+    def check_has_dicom_node(cls, values):
```

Comment:
don't need to change this but could maybe be more descriptive

```suggestion
    def check_io_node_has_destination(cls, values):
```

name right now suggests they should all have dicom nodes at first glance